package.json

```diff
@@ -30,6 +31,7 @@
     "babel-jest": "^21.2.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Babel was throwing errors, had to add this package and import it before the `vue-material` package to fix